app/controllers/admin/users_controller.rb

```diff
@@ -16,6 +16,15 @@ class UsersController < Admin::ApplicationController
       email_body
     ].freeze
 
+    ATTIBUTES_FOR_CSV = %i[
+      id name username email registered_at
+    ].freeze
+
+    ATTRIBUTES_FOR_LAST_ACTIVITY = %i[
+      registered last_comment_at last_article_at latest_article_updated_at last_reacted_at profile_updated_at
+      last_moderation_notification last_notification_activity
+    ].freeze
```

Comment:
I did not want to simply do a `User.all` for performance reasons so instead I've selected the columns that we show in the CSV. However, `user.last_activity` depends on a number of columns, hence I've separated it out into two different constants to reflect that.